php/php.editor/src/org/netbeans/modules/php/editor/actions/FixDuplicateImportStmts.form

```diff
@@ -31,9 +31,10 @@
   </Properties>
   <AuxValues>
     <AuxValue name="FormSettings_autoSetComponentName" type="java.lang.Boolean" value="false"/>
-    <AuxValue name="FormSettings_generateFQN" type="java.lang.Boolean" value="true"/>
+    <AuxValue name="FormSettings_generateFQN" type="java.lang.Boolean" value="false"/>
```

Comment:
We are changing the default value here - no problem, I think. If there are any complaints, we could store it in the userdir so it is kept between individual invocations (if we are not doing it already :)